### PR TITLE
Added more qualifications

### PR DESCRIPTION
**Pull request by WillTheDeveloper**
Please review commits.

### CODE REVIEW

**Review by colsak — COMMENTED** [review]
`src/aqatechnical.cpp` has if statements but doesn't have the return values. Are you going to add them in another commit?

**Comment by WillTheDeveloper** [comment]
> `src/aqatechnical.cpp` has if statements but doesn't have the return values. Are you going to add them in another commit?

Yeah I am

**Review by colsak — CHANGES_REQUESTED** [review]
`src/header/aqatechnical.h` doesn't have an `#endif` at the end of the header file.

File: src/header/aqatechnical.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+#include "string"
+#include <iostream>
+
+namespace AQATechnicalN {
+    int L3FoundationTechnical(std::string pts); //D*, D, M, P
+    int L3Techincal1080(std::string pts); // D*D*D* -> PPP
+    int L3Technical540(std::string pts); // D*D* -> PP
+    int L3Technical720(std::string pts); // D*D* -> PP
+}
```
Comment (colsak) [inline src/header/aqatechnical.h]: MMm no `#endif`

Comment (WillTheDeveloper) [reply src/header/aqatechnical.h]: Just fixed it

**Review by colsak — APPROVED** [review]

**Pull request merged**